web/src/enterprise/campaigns/detail/CampaignActionsBar.tsx

```diff
@@ -104,16 +105,18 @@ export const CampaignActionsBar: React.FunctionComponent<Props> = ({
                         </>
                     ) : (
                         <>
-                            <button
-                                type="button"
-                                id="e2e-campaign-edit"
-                                className="btn btn-secondary mr-1"
-                                onClick={onEdit}
-                                disabled={actionsDisabled}
-                            >
-                                Edit
-                            </button>
-                            {!campaign.closedAt && (
+                            {!campaignClosed && (
+                                <button
+                                    type="button"
+                                    id="e2e-campaign-edit"
+                                    className="btn btn-secondary mr-1"
+                                    onClick={onEdit}
+                                    disabled={actionsDisabled}
+                                >
+                                    Edit
+                                </button>
+                            )}
+                            {!campaignClosed && (
                                 <CloseDeleteCampaignPrompt
                                     disabled={actionsDisabled}
                                     disabledTooltip="Cannot close while campaign is being created"
```

Comment:
```suggestion
                            {!campaignClosed && (
                            <>
                                <button
                                    type="button"
                                    id="e2e-campaign-edit"
                                    className="btn btn-secondary mr-1"
                                    onClick={onEdit}
                                    disabled={actionsDisabled}
                                >
                                    Edit
                                </button>
                                <CloseDeleteCampaignPrompt
                                    disabled={actionsDisabled}
                                    disabledTooltip="Cannot close while campaign is being created"
```

would need a closing fragment `</>` below but GitHub doesn't allow me to suggest on untouched lines ..